### PR TITLE
[TC-9114] Add tXML

### PR DESCRIPTION
**Pull request by marcmatt**
## Description

https://tradecloud.atlassian.net/browse/TC-9114

### Scope
This PR adds tXML. please review https://app.gitbook.com/o/-LNyIUZSvpZWZ81yEr7V/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/ENOJ2zwBudff1wuuvCVW/introduction/api/json-vs-xml and other pages.

### Submitter pre-flight check

- [x] Review your documentation
- [x] Add labels `needs reviewing`
- [x] Assign PR and JIRA ticket to 'Reviewer' (in 'Reviewing' state)

### Reviewer pre-flight check

- [ ] Review documentation in PR
- [ ] Remove label `needs reviewing`
- [ ] Merge the PR and remove the release for this branch from Gitbook
- [ ] Set Jira ticket to 'Released'

### CODE REVIEW

File: api/json-vs-xml.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+description: >-
+  Choose between JSON and XML in the `api-connector` and `order-webhook-connector`
+---
+
+# JSON versus XML
+
+The Tradecloud API supports default a proprietary JSON format, but for some API endpoints a proprietary XML format is supported.
```
Comment (roy-tc) [inline api/json-vs-xml.md:8]:
```suggestion
The Tradecloud API by default supports a proprietary JSON format, but for some API endpoints a proprietary XML format is supported.
```

File: api/json-vs-xml.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## JSON
+
+The Tradecloud API supports default a proprietary Tradecloud [JSON format](requests.md#json-body), called `TSON`.
```
Comment (roy-tc) [inline api/json-vs-xml.md:12]:
```suggestion
The Tradecloud API by default supports a proprietary Tradecloud [JSON format](requests.md#json-body), called `TSON`.
```

File: api/json-vs-xml.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+All `api-connector`, `order-search`, `shipment-search`, `order-webhook-connector` and `shipment-webhook-connector` API endpoints support `TSON`.
```
Comment (roy-tc) [inline api/json-vs-xml.md:14]: Do we really want to use `TSON`? imo it's funny, but not adding any value.
If anything, I'd say we adher to https://datatracker.ietf.org/doc/html/rfc7159, so it's just JSON.

For XML it's a bit different, as the notion of a "schema" is an intrinsic part of the syntax. In JSON, this is not the case afaik.

File: api/json-vs-xml.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## XML
+
+The Tradecloud API supports a proprietary Tradecloud [XML format](requests.md#xml-body), called `tXML`, which is a 1-on-1 translation of the `TSON` format:
+
+Only the [Send order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendOrderByBuyerRoute) and [Send simple order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendSimpleOrderByBuyerRoute) endpoints support `tXML` at this moment.
+
+You can see an XML example by selecting "application/xml" in the "Parameter content type" dropdown, under the "Example Value" in above API specifications.
```
Comment (roy-tc) [inline api/json-vs-xml.md:22]: Nit: may be add a screenshot with some arrows?

File: api/requests.md [hunk 5]
```diff
@@ -118,7 +119,34 @@ JSON is a standard published as [RFC 8259](https://tools.ietf.org/html/rfc8259)
...
+
+#### tXML
+
+The Tradecloud API supports a proprietary XML format, called `tXML`, which is a 1-on-1 translation of the proprietary `TSON` format:
+
+A POST request has a `tXML` body \(payload\), such as a purchase order sent to the `api-connector` service `order` method:
+
+[`POST /api-connector/order`](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendOrderByBuyerRoute)
+
+```markup
```
Comment (roy-tc) [inline api/requests.md:135]:
```suggestion
```xml
```

**Review by roy-tc — COMMENTED** [review]
Looks good! Some small suggestions..

File: api/json-vs-xml.md (hunk 3 above)
Comment (marcmatt) [reply api/json-vs-xml.md]: I removed TSON, but just like our proprietary XML format, it is our proprietary JSON based format, its not just JSON.

File: api/json-vs-xml.md (hunk 4 above)
Comment (marcmatt) [reply api/json-vs-xml.md:22]: As far as I understand
- I have to update the image in a change request and merge it
- then reference it like `Sending a Login request`

Comment (marcmatt) [reply api/json-vs-xml.md:22]: Ah no, I can add it directly to .gitbook/assists_ using the PR

Comment (roy-tc) [reply api/json-vs-xml.md:22]: @marcmatt indeed, see `swagger-ui.md` for examples.

File: api/json-vs-xml.md (hunk 3 above)
Comment (roy-tc) [reply api/json-vs-xml.md]: "proprietary JSON" sounds perfect to me.

File: api/json-vs-xml.md (hunk 4 above)
Comment (marcmatt) [reply api/json-vs-xml.md]: Added

**Pull request merged**